swift/apple/FirezoneKit/Sources/FirezoneKit/Helpers/Log.swift

```diff
@@ -127,12 +139,6 @@ private final class LogWriter {
 
     jsonData.append(newLineData)
 
-    workQueue.async {
-      do {
-        try jsonData.write(to: self.logFileURL, options: .atomic)
-      } catch {
-        self.logger.error("Could not write LogEntry! \(error)")
-      }
-    }
+    workQueue.async { self.handle.write(jsonData) }
```

Comment:
And the newline either gets added by `write` or it's already in `jsonData`, right?

Reply:
It's appended to jsonData in the write() call.

Reply:
Here: https://github.com/firezone/firezone/pull/4633/files#diff-9ff9afd32165d8c6986539221e9fcff2aafa93f0be374fdf0660cfd3aedc7ea4R134